tools/clang/tools/dxcompiler/dxclinker.cpp

```diff
@@ -131,7 +131,7 @@ DxcLinker::RegisterLibrary(_In_opt_ LPCWSTR pLibName, // Name of the library.
     CComPtr<IMalloc> pMalloc;
     CComPtr<AbstractMemoryStream> pDiagStream;
 
-    IFT(CoGetMalloc(1, &pMalloc));
+    IFT(DxcCoGetMalloc(1, &pMalloc));
```

Comment:
I think we should get rid of this, and just use `DxcGetThreadMallocNoRef()` in place of pMalloc below.

Reply:
Done.